### DISCUSSION TITLE
How to dump the atomic energy from lammps

### DISCUSSION THREAD

**Discussion by cjxxjc729** [deepmodeling/deepmd-kit#2322] [Q&A]
Hellow everyone. I wonder how to dump the atomic energy from lammps. Which keyword should we use in lammps input file?

**Comment by njzjz** [comment]
Atomic energy should be printed by [compute pe/atom command](https://docs.lammps.org/compute_pe_atom.html).

However, I remember the atomic energy is not passed to lammps in pair_style deepmd. Maybe you could confirm the situation.

**Reply by cjxxjc729** [reply]
Yes. compute pe/atom command works. And I verify that by DeepPot. Thanks a lot!
